content/en/docs/ops/diagnostic-tools/istioctl/index.md

```diff
@@ -20,7 +18,34 @@ The [`istioctl`](/docs/reference/commands/istioctl) tool is a configuration comm
 `istioctl` only has auto-completion enabled for non-deprecated commands.
 {{< /tip >}}
 
-### Get an overview of your mesh
+## Before you begin
+
+We recommend you use an `istioctl` version that is the same version as your Istio control plane. Using matching versions helps avoid unforeseen issues.
+
+{{< tip >}}
+If you have already [downloaded the Istio release](/docs/setup/getting-started/#download), you should
+already have `istioctl` and do not need to install it again.
+{{< /tip >}}
+
+## Install {{< istioctl >}}
+
+Install the `istioctl` binary with `curl`:
+
+1. Download the latest release with the command:
+
+    {{< text bash >}}
+    $ curl -sL https://raw.githubusercontent.com/istio/istio/release-{{< istio_version >}}/release/downloadIstioCtl.sh | sh -
```

Comment:
This is causing a problem for the linter. The linter wants you to use

```suggestion
    $ curl -sL {{< github_file >}}/release-{{< istio_version >}}/release/downloadIstioCtl.sh | sh -
```

But that ends up making a link that doesn't work for cut & paste.

I'll find a solution in a bit...

Reply:
@linsun @howardjohn @ericvn @sdake @frankbu 

Right now, to download Istio, we tell users to use:

    $ curl -L https://git.io/getLatestIstio | ISTIO_VERSION={{< istio_full_version >}} sh -

which points to 

https://raw.githubusercontent.com/istio/istio/master/release/downloadIstioCandidate.sh

To fix the lint error, I propose replacing the two uses with:

https://istio.io/downloadIstio
https://istio.io/downloadIstoctl

These links would be automatically updated to always point to the scripts from the current release branch. We would abandon the exiting git.io URL for download.

WDYT?

Reply:
That is a great idea.... I would keep the old git.io url for istio binary for backwards compatible for a release or two.

Reply:
We can't actually remove the link from git.io, it's a read-only system. We just stop talking about it.